server/zanata-war/src/test/java/org/zanata/adapter/SubtitleAdapterTest.java

```diff
@@ -33,126 +36,163 @@
 import static org.assertj.core.api.Assertions.assertThat;
 
 /**
- * @author djansen <a href="mailto:djansen@redhat.com">djansen@redhat.com</a> on 6/02/17.
+ * @author djansen <a href="mailto:djansen@redhat.com">djansen@redhat.com</a>
  * @// TODO: 6/02/17 test ids
  */
+@RunWith(Enclosed.class)
 public class SubtitleAdapterTest {
 
-    private SubtitleAdapter adapter;
-    private final String[] fileTypes = new String[]{"srt", "vtt", "sbt", "sub"};
+    public static abstract class SubtitleAdapterSuper {
+        private static SubtitleAdapter adapter;
```

Comment:
- Package private should be enough; no need for public.
- The adapter field doesn't need to be static.
- Class name: how about AbstractSubtitleTest? It's not important though, especially if it becomes non-private.